nav/nav_goal/src/tolerance_parameters.py

```diff
@@ -0,0 +1,8 @@
+# parameters for tolerance (xyz in meters and orientation in radians)
+x_tol = 1.0
+y_tol = 1.0
+z_tol = 1.0
+yaw_tol = 3.14
+pitch_tol = 3.14
+roll_tol = 3.14
```

Comment:
I would just make these tolerances variables in `movebase_client.py` and not have another `.py` file just for constants

Reply:
Resolved